addons/sale_timesheet/__manifest__.py

```diff
@@ -16,7 +16,6 @@
     'depends': ['sale_management', 'hr_timesheet'],
```

Comment:
```suggestion
     'depends': ['sale_project', 'hr_timesheet'],
```
Why not this? I mean, imagine you have installed `sale_timesheet`, but then you try to uninstall `sale_project`. When uninstalling  `sale_project`, should `sale_timesheet` be also automatically uninstalled, or not? If not, then close this comment.